acquia/acquia_cms
# ACMS-3354: Remove 10.2.x DIT patches from Acquia CMS Common module.

**Motivation**
Fixes #[3354](https://acquia.atlassian.net/browse/ACMS-3354)

**Proposed changes**
Remove Drupal core patches as available in 10.2.x core.

**Alternatives considered**
NA

**Testing steps**
- Checkout branch ACMS-3354
- Install dependencies using `composer install`
- verify patches applied successfully.

**Merge requirements**
